README.md

```diff
@@ -284,15 +284,15 @@ Authentication optional, will return multiple articles, ordered by most recent
 
```

Comment:
@djallas As I see, you changed the readme.md file yet the routes are still the same, and on my behalf, I think routes should be more descriptives like if I want to fetch an article by slug, then it is better if the route is : ``/articles/:slug`` than just ``/:slug`` because it maybe confusing

Reply:
@rwajon thanks for this, It's was by mistake, Actually It happened when I was replacing a pattern in my code. I fix this.